### PR TITLE
update aws test github actions

### PR DESCRIPTION
**Pull request by ggabernet**

### CODE REVIEW

File: .github/workflows/awstest.yml [hunk 1]
```diff
@@ -10,12 +10,9 @@ on:
...
 jobs:
-    Awstest:
-        runs-on: ${{ matrix.os }}
-        strategy:
-            matrix:
-                os: ['ubuntu-latest']
-                python-version: ['3.7']
+    run-awstest:
+        runs-on: ubuntu-latest
+        if: github.repository == 'nf-core/awsmegatests'
```
Comment (FriederikeHanssen) [inline .github/workflows/awstest.yml:15]: Does this workflow script then only run, if it is in the repo `nf-core/awsmegatests`. Does this line need to be updated when running tests for a different pipeline?

File: .github/workflows/awstest.yml [hunk 2]
```diff
@@ -25,10 +22,18 @@ jobs:
...
+            AWS_JOB_QUEUE: ${{ secrets.AWS_JOB_QUEUE }}
+            AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
           run: |
-            aws batch submit-job --region eu-west-1 --job-name nf-core-awstest --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow --container-overrides '{"command": ["nf-core/rnaseq", "-profile test --outdir s3://nf-core-awsmegatests/rnaseq/results -w s3://nf-core-awsmegatests/rnaseq/work -with-tower"],"environment": [{"name": "TOWER_ACCESS_TOKEN","value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
+            aws batch submit-job \
+            --region eu-west-1 \
+            --job-name nf-core-awstest \
+            --job-queue $AWS_JOB_QUEUE \
+            --job-definition $AWS_JOB_DEFINITION \
+            --container-overrides '{"command": ["nf-core/rnaseq", "-r dev -profile test --outdir s3://'"${AWS_S3_BUCKET}"'/awsmegatests/results_'"${GITHUB_SHA}"' -w s3://'"${AWS_S3_BUCKET}"'/awsmegatests/work_'"${GITHUB_SHA}"' -with-tower"],"environment": [{"name": "TOWER_ACCESS_TOKEN","value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```
Comment (FriederikeHanssen) [inline .github/workflows/awstest.yml:37]: Maybe we should add a note in the document, that the user needs to replace `rnaseq` with their pipeline name

Comment (ggabernet) [reply .github/workflows/awstest.yml:37]: that's part of the pipeline template now, and it gets automatically replaced so it should be ok :)

File: .github/workflows/awstest.yml (hunk 1 above)
Comment (ggabernet) [reply .github/workflows/awstest.yml:15]: yes, that is then updated to the pipeline name. As these workflows are part of the pipeline template now, the cookie cutter automatically updates the names

**Pull request merged**